.github/workflows/push.yml

```diff
@@ -90,7 +89,9 @@ jobs:
           if [ $branch == "master" ] || [ ${array[1]} == "tags" ] || [[ $branch =~ ^v[0-9]*.[0-9]*.x$ ]]
           then
             source env-vars
-            docker push ${{ secrets.REPONAME }}/${GRAPHQL_SERVER_IMAGE}:${IMG_TAG}
+            cd litmus-portal && make docker.buildx
+            cd graphql-server
+            docker buildx build --file build/Dockerfile --progress plane --push --no-cache --platform linux/arm64,linux/amd64 --tag ${{ secrets.REPONAME }}/${GRAPHQL_SERVER_IMAGE}:${IMG_TAG} .
```

Comment:
Only 64 bit arch ? arm v7 or 32 bits can be supported too @uditgaurav

Reply:
We have added support for ARM64 with default variant 6  as of now for all other components. We can add any additional platform as per the community request as this will increase the build time @ishangupta-ds .